testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -80,3 +80,7 @@ Feature: Disable and re-enable monitoring of the server
     And I should see a list item with text "Taskomatic (Java JMX)" and a success bullet
     And I should see a list item with text "Tomcat (Java JMX)" and a success bullet
     And I should not see a "Restarting Tomcat and Taskomatic is needed for the configuration changes to take effect." text
+    Then port "3333" should be closed
```

Comment:
And

Reply:
Thank you, I missed that after copy pasting. Fixed.